kvmagent/kvmagent/plugins/vm_metadata/rw.py

```
@@ -0,0 +1,738 @@
+
+def _determine_op_type(fd, header, new_payload):
+    """Determine PENDING_CONFIG_UPDATE or PENDING_STORAGE_CHANGE."""
+    try:
+        active_slot = _read_active_slot(fd, header)
+        if active_slot.valid and active_slot.payload:
+            if _storage_topology_changed(active_slot.payload, new_payload):
+                return PENDING_STORAGE_CHANGE
+            else:
+                return PENDING_CONFIG_UPDATE
+    except Exception:
+        pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)